py-polars/polars/internals/expr/expr.py

```diff
@@ -3153,64 +3153,30 @@ def wrap_f(x: pli.Series) -> pli.Series:  # pragma: no cover
 
     def flatten(self) -> Expr:
         """
-        Alias for :func:`explode`.
+        Flatten a list or string column after a groupby or window operation.
```

Comment:
It can also be flattened in select context. I think we should keep it a generic description.

Reply:
Updated!

Reply:
Thanks! :)